allocator.py

```diff
@@ -0,0 +1,92 @@
+from people import Fellow, Staff
+from room import Office, LivingSpace
+import random
+
+class Dojo(object):
+    def __init__(self):
+        self.room_name = []
+        self.offices = {}
+        self.livings = {}
+
+        self.staff_list = []
+        self.fellow_list = []
+
+    def create_room(self, is_office, room_type):
+        rooms = room_type
+        # check if its staff or fellow and create person by calling staff and fellow objects
+
+        if is_office:
+            for room in rooms:
+                create_office = Office(room)
+                self.offices[create_office.name] = []
+                print("Office ", room, " has been created")
+
+        else:
+            for room in rooms:
+                create_living = LivingSpace(room)
+                self.livings[create_living.name] = []
+                print("Living space ", room, " has been created")
+
+
+    def create_person(self, fname, lname, is_fellow, need_room='No'):
+        names = fname + " " + lname
+        if is_fellow:
+            create_fellow = Fellow(names)
+            self.fellow_list.append(create_fellow)
+
+            if need_room == 'No':
+                need_room = False
+                self.assign_room(create_fellow, need_room)
+            else:
+                need_room = True
+                self.assign_room(create_fellow, need_room)
+
+        else:
+            create_staff = Staff(names)
+            self.staff_list.append(create_staff)
+            self.assign_room(create_staff, need_room)
```

Comment:
See above comment regarding `create_fellow`. Its the same case as `create_staff`